internal/types/subtypes/interceptor.go

```diff
@@ -298,7 +313,7 @@ func AttributeTransformerInterceptor(_ context.Context) grpc.UnaryServerIntercep
 		if reqMsg, ok := req.(proto.Message); ok {
 			if err := transformRequest(reqMsg); err != nil {
 				return nil, handlers.InvalidArgumentErrorf("Error in provided request.",
-					map[string]string{"attributes": "Attribute fields do not match the expected format."})
+					map[string]string{"attributes": fmt.Sprintf("Attribute fields do not match the expected format: %s.", parseErrorDetails(err))})
```

Comment:
I agree, I don't like the idea of parsing the error strings too because of how brittle it can become. I just wasn't sure how big of a change was acceptable for an issue fix without going through sprint planning. Because I feel like introducing custom error types will affect many files in the repository, which is why wanted to avoid such a large change without grooming a ticket for it. I will reimplement a long term solution for this over the parsing method.